src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -1251,6 +1251,94 @@ default List<Member> getMembersWithRoles(@Nonnull Collection<Role> roles)
     @Nonnull
     MemberCacheView getMemberCache();
 
+
+    /**
+     * Sorted {@link SnowflakeCacheView} of
+     * all cached {@link GuildScheduledEvent GuildScheduledEvents} of this Guild.
+     * <br>Scheduled events are sorted by their start time, and events that start at the same time
+     * are sorted by their snowflake ID.
+     *
+     * @return {@link SortedSnowflakeCacheView}
+     */
+    @Nonnull
+    SortedSnowflakeCacheView<GuildScheduledEvent> getScheduledEventCache();
+    
+    /**
+     * Gets a list of all {@link net.dv8tion.jda.api.entities.GuildScheduledEvent GuildScheduledEvents} in this Guild that have the same
+     * name as the one provided.
+     * <br>If there are no {@link net.dv8tion.jda.api.entities.GuildScheduledEvent GuildScheduledEvents} with the provided name,
+     * then this returns an empty list.
+     *
+     * @param  name
+     *         The name used to filter the returned {@link GuildScheduledEvent} objects.
+     * @param  ignoreCase
+     *         Determines if the comparison ignores case when comparing. True - case insensitive.
+     *
+     * @return Possibly-empty immutable list of all GuildScheduledEvent names that match the provided name.
+     */
+    @Nonnull
+    default List<GuildScheduledEvent> getScheduledEventsbyName(@Nonnull String name, boolean ignoreCase)
```

Comment:
Missing docs for null check on name

Reply:
Also the casing should be `getXyzByName` not `getXyzbyName`

Reply:
added and fixed